domain-cc/svc-cc-app/src/util/data/dc_lookup_table_mvp.json

```diff
@@ -0,0 +1,3 @@
+{
+  "5329": 9005, "6510": 9012, "6511": 9012, "6512": 9012, "6513": 9012, "6514": 9012, "6523": 9012, "6524": 9012, "6600": 9012, "6601": 9012, "6602": 9012, "6603": 9012, "6604": 9012, "6732": 6890, "6825": 9012, "6826": 9012, "6827": 9012, "6828": 9012, "6829": 9012, "6830": 9012, "6831": 9012, "6832": 9012, "6833": 9012, "6845": 9012, "6846": 9012, "6847": 9012, "7101": 8979, "8100": 9007, "9918": 8967, "7718": 8981, "7725": 8981
+}
```

Comment:
nit: newline at EOF

Reply:
won't fix, this is programmatically generated